test/conditions/boolean_test.exs

```diff
@@ -26,18 +27,18 @@ defmodule FiltrexConditionBooleanTest do
 
   test "encoding true value" do
     assert Filtrex.Encoder.encode(condition(true, "equals")) ==
-      %Filtrex.Fragment{expression: "flag = ?", values: [true]}
+      %Filtrex.Fragment{expression: "? = ?", values: [column_ref(:flag), true]}
```

Comment:
Similar to previous question, will this result in a string or something else?

Reply:
`column_ref` returns a macro AST for `s.column_name`.

```
{{:., [], [{:s, [], Filtrex.Util.Encoder}, :column_name]}, [], []}
```

The `AST` module unquotes these `values` into the final Ecto query fragment macro, resulting in `s.column_name`.